examples/php.ini

```diff
@@ -0,0 +1,2 @@
+display_errors = off
```

Comment:
I think it might be better to display errors. If we're assuming that these are just _demo_ projects, and people won't just copy and paste them to use in their production environments, do we think it'd be more useful to show the error, rather than having to find it in the logs?

Reply:
In .NET and Go (maybe others), we're currently displaying the error

Reply:
We could hide them here, but display them in demo project built in pipeline?

Reply:
The errors will continue to show in the terminal when running the example